glide.yaml

```diff
@@ -7,5 +7,6 @@ import:
 - package: github.com/dgrijalva/jwt-go
   version: v3.2.0
 - package: github.com/edgexfoundry/edgex-go
+  version: delhi
```

Comment:
This line still needs be changed to master as this PR is targeted master...

Reply:
addressed in this PR with another push.